### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by anurag6**

### CODE REVIEW

File: testing/run_unit_tests.sh [hunk 1]
```diff
@@ -6,13 +6,15 @@ TESTDIR=`dirname $SCRIPTPATH`
...
 
-PYTHONPATH=$BASEDIR/daq:$BASEDIR/mininet:$BASEDIR/faucet:$BASEDIR/forch $BASEDIR/venv/bin/coverage run \
+coverage erase
+
+PYTHONPATH=$BASEDIR/daq:$BASEDIR/mininet:$BASEDIR/faucet:$BASEDIR/forch coverage run \
     --source $BASEDIR/daq \
     -m unittest discover \
     -s $TESTDIR/unit/ \
-    -p "test_*.py"
+    -p "test_*.py" || exit 1
```
Comment (grafnu) [inline testing/run_unit_tests.sh:17]: exit 1 is redundant with -e option at top

File: testing/run_unit_tests.sh [hunk 2]
```diff
@@ -6,13 +6,15 @@ TESTDIR=`dirname $SCRIPTPATH`
 BASEDIR=`readlink -f $TESTDIR/..`
 cd $BASEDIR
 
-$BASEDIR/venv/bin/coverage erase
+source venv/bin/activate
 
-PYTHONPATH=$BASEDIR/daq:$BASEDIR/mininet:$BASEDIR/faucet:$BASEDIR/forch $BASEDIR/venv/bin/coverage run \
+coverage erase
+
+PYTHONPATH=$BASEDIR/daq:$BASEDIR/mininet:$BASEDIR/faucet:$BASEDIR/forch coverage run \
```
Comment (grafnu) [inline testing/run_unit_tests.sh:13]: set separately and export -- this is getting ugly as a prefix

File: testing/run_unit_tests.sh (hunk 1 above)
Comment (anurag6) [reply testing/run_unit_tests.sh]: Done.

File: testing/run_unit_tests.sh (hunk 2 above)
Comment (anurag6) [reply testing/run_unit_tests.sh]: Done

**Pull request merged**